### ISSUE TITLE
src dirs not showing in Project sidebar

### ISSUE THREAD

**Issue by roblg** [bazelbuild/intellij#328]
Java source directories appear to be completely disappearing from the `Project` view in the IntelliJ sidebar.

IntelliJ 2018.1.2 and 2018.1.4
ijwb: 2018.05.07.0.2

Project looks like:
```
foo.client:
  src/main/java/redfin/**/*.java
  BUILD
```

BUILD file:
```
java_library(
    name = "foo.client",
    srcs = glob(["src/main/java/**/*.java"]),
    visibility = ["//visibility:public"],
    deps = [ 
        # snip
    ],
)

```

This is a very simple example, but this appears to be happening in all of our packages after upgrading to IntelliJ 2018.x/new bazel plugin. I was originally using the old "external" (`../main-00`) form of project dir, and I created a totally new IntelliJ project using the new default (`./.project`) and it still has the issue after a full sync.

**However** when I _first_ imported the workspace and created the new project while I was attempting to debug, I _did_ briefly see the directory structure I expected. I noticed at that time that the difference between the old .blaze/modules/.workspace.iml file and the new one was the presence of `packagePrefix` attributes on each of the `sourceFolder` entries.

e.g., old and busted:

```
<sourceFolder url="file://$MODULE_DIR$/../../../foo.client/src/main/java" isTestSource="false" />
```

new, temporarily worked
```
<sourceFolder url="file://$MODULE_DIR$/../../../foo.client/src/main/java" isTestSource="false" packagePrefix="<a.path.that.makes.sense>" />
```

When the `sync` finished in the new project, the `packagePrefix` attrs were gone in the .workspace.iml file, and I'm back to no source dirs showing up.

**Comment by roblg** [comment]
Oh, follow-up: the sources still appear to be being processed. I can see them in the "Project Files" sidebar pane (everything in a flat list), and I can find them with "Open File" and "Open Class" shortcuts.

**Comment by plumpy** [comment]
This should be fixed in Bazel plugin version 2018.05.21, just uploaded today.
